### PR TITLE
vim-patch:partial 113cb513f76d

### PR DESCRIPTION
**Pull request by clason**
Update runtime files
https://github.com/vim/vim/commit/113cb513f76d8866cbb6dc85fa18aded753e01da

skip doc/eval.txt
skip doc/insert.txt

### CODE REVIEW

File: runtime/doc/syntax.txt [hunk 1]
```diff
@@ -4768,16 +4782,28 @@ in their own color.
...
+			For changing the appearance of specific colors, you
+			can redefine a color name before loading the scheme.
+			The desert scheme uses the khaki color for the cursor.
+			To use a darker variation of the same color: >
+
+				let v:colornames['khaki'] = '#bdb76b'
+				colorscheme desert
+<
+			For further customization, such as changing
+			|:highlight-link| associations, use another name, e.g.
```
Comment (seandewar) [inline runtime/doc/syntax.txt:4795]: Need https://github.com/vim/vim/commit/e30d10253fa634c4f60daa798d029245f4eed393

File: runtime/doc/indent.txt [hunk 2]
```diff
@@ -604,26 +604,25 @@ CLOJURE					*ft-clojure-indent* *clojure-indent*
 Clojure indentation differs somewhat from traditional Lisps, due in part to
 the use of square and curly brackets, and otherwise by community convention.
 These conventions are not universally followed, so the Clojure indent script
-offers a few configurable options, listed below.
+offers a few configuration options.
+
+(If the current Vim does not include |searchpairpos()|, the indent script falls
+back to normal 'lisp' indenting, and the following options are ignored.)
```
Comment (seandewar) [inline runtime/doc/indent.txt:610]: Nvim always has `searchpairpos`.

**Review by seandewar — COMMENTED** [review]
Missing https://github.com/vim/vim/commit/113cb513f76d8866cbb6dc85fa18aded753e01da#diff-f17e0dd888d6e8cd63e14b598ed3a141e59870e04a35701b66c93ddb175f2a57L537-R537

File: runtime/doc/syntax.txt [hunk 3]
```diff
@@ -4768,16 +4782,18 @@ in their own color.
...
-<			Before the color scheme will be loaded the
-			|ColorSchemePre| autocommand event is triggered.
-			After the color scheme has been loaded the
-			|ColorScheme| autocommand event is triggered.
+<			Before the color scheme will be loaded all default
+			color list scripts (`colors/lists/default.vim`) will
+			be executed and then the |ColorSchemePre| autocommand
+			event is triggered.  After the color scheme has been
+			loaded the |ColorScheme| autocommand event is
+			triggered.
```
Comment (seandewar) [inline runtime/doc/syntax.txt:4796]: Needs `v:colornames` patch

File: runtime/doc/syntax.txt [hunk 4]
```diff
@@ -4800,7 +4816,9 @@ in their own color.
 
 :hi[ghlight] [default] {group-name} {key}={arg} ..
 			Add a highlight group, or change the highlighting for
-			an existing group.
+			an existing group. If a given color name is not
+			recognized, each `colors/lists/default.vim` found on
+			|'runtimepath'| will be loaded.
```
Comment (seandewar) [inline runtime/doc/syntax.txt:4821]: Needs `v:colornames` patch

**Review by seandewar — COMMENTED** [review]
(Also) Skipped `usr_06.txt` (needs `v:colornames`): https://github.com/vim/vim/commit/113cb513f76d8866cbb6dc85fa18aded753e01da#diff-c8ca485f457a3d0ea142c0371d3e97ffdb1368cb8cbd69199a5a39decbc0e82c

**Review by seandewar — APPROVED** [review]

File: runtime/doc/options.txt [hunk 5]
```diff
@@ -3703,9 +3705,10 @@ A jump table for the options with a short description can be found at |Q_op|.
 						*'list'* *'nolist'*
 'list'			boolean	(default off)
 			local to window
-	List mode: Show tabs as CTRL-I is displayed, display $ after end of
-	line.  Useful to see the difference between tabs and spaces and for
-	trailing blanks.  Further changed by the 'listchars' option.
+	List mode: By default show tabs as CTRL-I is displayed, display $
+	after end of line.  Useful to see the difference between tabs and
+	spaces and for trailing blanks.  Further changed by the 'listchars'
+	option.
```
Comment (seandewar) [inline runtime/doc/options.txt:3711]: This description is actually inaccurate, but #16254 should resolve that

Comment (clason) [reply runtime/doc/options.txt:3711]: Went ahead and merged, then, so @dundargoc can rebase.

**Pull request merged**